frontend/packages/kubevirt-plugin/src/models/templates/vm-template-yaml.ts

```diff
@@ -10,20 +10,21 @@ metadata:
   name: vm-template-example
   labels:
     kubevirt.io/os: fedora27
-    miq.github.io/kubevirt-is-vm-template: 'true'
     template.kubevirt.io/type: vm
+    flavor.template.kubevirt.io/Custom: 'true'
+    os.template.kubevirt.io/fedora27: 'true'
+    vm.kubevirt.io/template-namespace: openshift
+    workload.template.kubevirt.io/server: 'true'
   annotations:
-    description: VM template example
-    iconClass: icon-fedora
-    tags: 'kubevirt,ocp,template,linux,virtualmachine'
+    name.os.template.kubevirt.io/fedora27: Fedora 27
 objects:
   - apiVersion: kubevirt.io/v1alpha3
     kind: VirtualMachine
     metadata:
-      creationTimestamp: null
       labels:
-        kubevirt-vm: 'vm-\${NAME}'
-        kubevirt.io/os: fedora27
+        app: '\${NAME}'
+        vm.kubevirt.io/template.revision: '1'
+        vm.kubevirt.io/template.version: v0.6.2
```

Comment:
Can we detect this version instead? The system might not have this or any other version of common templates.